src/components/playback/playbackmanager.js

```diff
@@ -3219,7 +3211,7 @@ class PlaybackManager {
             return player.currentTime();
         }
 
-        return this.getCurrentTicks(player);
+        return this.getCurrentTicks(player) / 10000;
```

Comment:
Reworking `getCurrentTicks` seems better than this (Converting this to `getCurrentTime` or removing it completely)